docs/integrations/send-logs-to-elasticsearch.rst

```diff
@@ -0,0 +1,58 @@
+Send logs to Elasticsearch®
+======================================
+
+You can store logs from one of your Aiven services in an external Elasticsearch service.
+
+You'll need the following values for the connection:
+
+============================     ==========================================================================================================
+Variable                         Description
+============================     ==========================================================================================================
+``ELASTICSEARCH_USER``           User name to access the Elasticsearch service.
+``ELASTICSEARCH_PASSWORD``       Password to access the Elasticsearch service.
+``ELASTICSEARCH_HOST``           HTTPS service host of your external Elasticsearch service.
+``ELASTICSEARCH_PORT``           Port to use for the connection.
+``CA_CERTIFICATE``               CA certificate in PEM structure (if necessary).
+``CONNECTION_NAME``              Name of your choosing for this external connection, that will be used with Aiven services.
+============================     ==========================================================================================================
+
+Create external Elasticsearch integration
+-------------------------------------------
+
+Start by setting up an external service integration for Elasticsearch.
+
+1. Log in to the Aiven web console.
+#. Navigate to **Service Integration** from the menu on the left.
+#. You'll see a list of external services you can integrate with Aiven.
+#. Select **External Elasticsearch** from the list.
+#. Press **Add new endpoint**.
+#. Set a preferred *endpoint name*, we'll call it ``CONNECTION_NAME`` later.
+#. In the connection URL field set the connection string in a format ``https://ELASTICSEARCH_USER:ELASTICSEARCH_PASSWORD@ELASTICSEARCH_HOST:ELASTICSEARCH_PORT``, using your own values for those parameters.
+#. Set desired index prefix, that doesn't overlap with any of already existing indexes in your Elasticsearch service.
```

Comment:
My question is: are we going to create a log per day? Why are we setting the prefix and not the entire index name?